rest-app/src/main/java/com/epam/repair/rest/controller/StreetController.java

```diff
@@ -3,11 +3,13 @@
 import com.epam.repair.model.Street;
 import com.epam.repair.service.StreetService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
+@CrossOrigin
```

Comment:
What for?

Reply:
Web application can not work with JSON with secure cross-origin requests

Reply:
Hi Alex, 
Method or controller CrossOrigin annotation is not flexible if you need a secure this application in a feature. Please review the possibility to create a global CORS configuration for the whole application in one place.